NCS.DSS.EmploymentProgression/PatchEmploymentProgression/Service/EmploymentProgressionPatchTriggerService.cs

```diff
@@ -0,0 +1,151 @@
+﻿using DFC.GeoCoding.Standard.AzureMaps.Model;
+using DFC.JSON.Standard;
+using NCS.DSS.EmploymentProgression.Cosmos.Provider;
+using NCS.DSS.EmploymentProgression.Models;
+using NCS.DSS.EmploymentProgression.ServiceBus;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace NCS.DSS.EmploymentProgression.PatchEmploymentProgression.Service
+{
+    public class EmploymentProgressionPatchTriggerService : IEmploymentProgressionPatchTriggerService
+    {
+        private readonly IJsonHelper _jsonHelper;
+        private readonly IDocumentDBProvider _documentDbProvider;
+        private readonly IServiceBusClient _serviceBusClient;
+
+        public EmploymentProgressionPatchTriggerService(IJsonHelper jsonHelper, IDocumentDBProvider documentDbProvider, IServiceBusClient serviceBusClient)
+        {
+            _jsonHelper = jsonHelper;
+            _documentDbProvider = documentDbProvider;
+            _serviceBusClient = serviceBusClient;
+        }
+
+        public async Task<string> GetEmploymentProgressionForCustomerToPatchAsync(Guid customerId, Guid employmentProgressionId)
+        {
+            var employmentProgressionAsString = await _documentDbProvider.GetEmploymentProgressionForCustomerToPatchAsync(customerId, employmentProgressionId);
+
+            return employmentProgressionAsString;
+        }
+
+        public string PatchEmploymentProgressionAsync(string employmentProgressionAsJson, EmploymentProgressionPatch employmentProgressionPatch)
```

Comment:
Not consistent with the other resources. Patch had its own class to handle patching